src/features/accounts/data/account-result.ts

```diff
@@ -31,36 +30,38 @@ const getAccountResult = async (address: Address) => {
   }
 }
 
-const syncAssociatedDataAndReturnAccountResultAtom = atom(null, async (get, set, address: Address) => {
-  const accountResult = await getAccountResult(address)
-  const assetResults = get(assetResultsAtom)
-  const applicationResults = get(applicationResultsAtom)
+const syncAssociatedDataAndReturnAccountResultAtom = atom(null, (get, set, address: Address) => {
+  return atom(async () => {
```

Comment:
To keep this close to the previous behaviour, can we keep the write only atom as async and perform all evaluations inside that. We can then make the returned atom a simple `atom(() => accountResult}`